src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateAssemblyInfo.targets

```diff
@@ -32,6 +32,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <GenerateAssemblyProductAttribute Condition="'$(GenerateAssemblyProductAttribute)' == ''">true</GenerateAssemblyProductAttribute>
     <GenerateAssemblyTitleAttribute Condition="'$(GenerateAssemblyTitleAttribute)' == ''">true</GenerateAssemblyTitleAttribute>
     <GenerateAssemblyVersionAttribute Condition="'$(GenerateAssemblyVersionAttribute)' == ''">true</GenerateAssemblyVersionAttribute>
+    <GenerateAssemblyMetadataAttribute Condition="'$(GenerateAssemblyMetadataAttribute)' == '' and !('$(TargetFrameworkIdentifier)' == '.NETFramework' and $([MSBuild]::VersionLessThan($(TargetFrameworkVersion), 4.5)))">true</GenerateAssemblyMetadataAttribute>
```

Comment:
Rename this to a less generic name.  We are using the generic `AssemblyMetadataAttribute` to store a specific piece of data (the repository URL).

I thingk `GenerateRepositoryUrlAttribute` is probably a good name.